cmd/libs/go2idl/client-gen/generators/fake/generator_fake_for_group.go

```diff
@@ -63,7 +63,7 @@ func (g *genFakeForGroup) GenerateType(c *generator.Context, t *types.Type, w io
 		"group":      g.group,
 		"Group":      namer.IC(g.group),
 		"Fake":       c.Universe.Type(types.Name{Package: pkgTestingCore, Name: "Fake"}),
-		"RESTClient": c.Universe.Type(types.Name{Package: pkgRESTClient, Name: "RESTClient"}),
+		"RESTClient": c.Universe.Type(types.Name{Package: pkgRESTClient, Name: "RESTClientInterface"}),
```

Comment:
go style is to omit redundant parts, so restclient.Interface should be the type (not restclient.RESTClientInterface).